src/Bridge/Doctrine/Common/Util/IdentifierManagerTrait.php

```diff
@@ -74,7 +76,13 @@ private function normalizeIdentifiers($id, ObjectManager $manager, string $resou
 
             $identifier = null === $identifiersMap ? $identifierValues[$i] ?? null : $identifiersMap[$propertyName] ?? null;
             if (null === $identifier) {
-                throw new PropertyNotFoundException(sprintf('Invalid identifier "%s", "%s" was not found.', $id, $propertyName));
+                $resourceShortName = '';
```

Comment:
Couldn't we replace this variable with the exception message (and concatenate the suffix in the condition)?